cns/wireserver/net.go

```diff
@@ -79,3 +123,10 @@ func calculateGatewayIP(cidr string) (net.IP, error) {
 	}
 	return gw, nil
 }
+
+func macAddressesEqual(macAddress1, macAddress2 string) bool {
+	macAddress1 = strings.ToLower(strings.ReplaceAll(macAddress1, ":", ""))
+	macAddress2 = strings.ToLower(strings.ReplaceAll(macAddress2, ":", ""))
+
+	return macAddress1 == macAddress2
+}
```

Comment:
nit: in what scenario would directly comparing two MAC addresses string doesn't work here? Why do we need to remove all the colons and convert them all to lowercases?

Reply:
the format for both is a bit different, one we get from cns goal state & another one is what we get from wireserver client